### DISCUSSION TITLE
How to rehandle qr code link?

### DISCUSSION THREAD

**Discussion by clankw33dmuffin** [tdlight-team/tdlight-java#162] [Q&A]
![image](https://github.com/tdlight-team/tdlight-java/assets/42200289/5d20299e-4add-4119-ae00-7a8fc1db5bee)
how i can save this link to variable?

**Comment by cavallium** [comment]
Unfortunately, the included CLI console login is not suitable for this use. You should create a custom AuthenticationSupplier.
